apps/frontend-manage/src/components/questions/QuestionEditModal.tsx

```diff
@@ -968,26 +986,32 @@ function QuestionEditModal({
                   {questionType === QuestionType.Numerical && (
                     <div>
                       <div className="w-full">
-                        <div className="flex flex-row items-center gap-2">
+                        <div className="flex flex-row items-center gap-2 mb-2">
                           <div className="font-bold">
                             {t('shared.generic.min')}:{' '}
                           </div>
-                          <FastField
+                          <FormikNumberField
```

Comment:
is there any performance impact because of this change? we had used fast field to counteract the lagging in inputs. but certainly makes sense to migrate to our own fields for this

Reply:
yes, there will probably be a small performance impact due to this change as the field will probably be validated more often. On the other hand, replacing it with our custom component results in a significantly more rigorous validation at the moment, wherefore the trade-off seemed to make sense.
The lagging was probably mainly caused by the editor component revalidating its content on every render, which seems like a minor problem here considering the length of the input?